pyarrowfs_adlgen2/core.py

```diff
@@ -32,6 +34,55 @@ def _parse_azure_ts(last_modified):
         return last_modified
 
 
+@dataclasses.dataclass
+class Timeouts:
+    """Timeouts passed to azure.storage.filedatalake operations
+
+    The value of these are provided as the timeout kwarg to the
+    corresponding object in azure.storage.filedatalake. Timeout
+    units are in seconds."""
+
+    file_client_timeout: typing.Optional[int]
```

Comment:
It's surprising to me that these are `int`, but that's what the SDK says for its timeout params 🤷